Formula/d/deskflow.rb

```diff
@@ -1,33 +1,32 @@
-class SynergyCore < Formula
-  desc "Synergy, the keyboard and mouse sharing tool"
-  homepage "https://symless.com/synergy"
-  url "https://github.com/symless/synergy/archive/refs/tags/1.15.1+r1.tar.gz"
-  version "1.15.1"
-  sha256 "42fbf26c634d2947c7efc45da8c9a153387bcdcb19c1102a4f7c4e95aad5c708"
+class Deskflow < Formula
+  desc "Network keyboard and mouse sharing tool"
+  homepage "https://deskflow.org"
+  url "https://github.com/deskflow/deskflow/archive/refs/tags/v1.17.0.tar.gz"
+  sha256 "5ba7fd70ca13f909ef23d6a5db250ebfae1e1f6c2ac89daf8d4cb3adc9e53538"
 
-  # The synergy-core/LICENSE file contains the following preamble:
+  # The deskflow/LICENSE file contains the following preamble:
   #   This program is released under the GPL with the additional exemption
   #   that compiling, linking, and/or using OpenSSL is allowed.
   # This preamble is followed by the text of the GPL-2.0.
   #
-  # The synergy-core license is a free software license but it cannot be
+  # The deskflow license is a free software license but it cannot be
   # represented with the brew `license` statement.
   #
   # The GitHub Licenses API incorrectly says that this project is licensed
   # strictly under GPLv2 (rather than GPLv2 plus a special exception).
-  # This requires synergy-core to appear as an exception in:
+  # This requires deskflow to appear as an exception in:
   #   audit_exceptions/permitted_formula_license_mismatches.json
   # That exception can be removed if the nonfree GitHub Licenses API is fixed.
   license :cannot_represent
-  head "https://github.com/symless/synergy-core.git", branch: "master"
+  head "https://github.com/deskflow/deskflow.git", branch: "master"
 
   # This repository contains old 2.0.0 tags, one of which uses a stable tag
   # format (`v2.0.0-stable`), despite being marked as "pre-release" on GitHub.
   # The `GithubLatest` strategy is used to avoid these old tags without having
   # to worry about missing a new 2.0.0 version in the future.
   livecheck do
     url :stable
-    regex(/[^"' >]*?v?(\d+(?:\.\d+)+)[^"' >]*?/i)
+    regex(/^v\d+.\d+.\d+/i)
     strategy :github_latest
   end
 
```

Comment:
need to remove the whole bottle block per formula rename

Reply:
are you talking about the `bottle do` section ?